app/views/admin/snap_applications/index.html.erb

```diff
@@ -50,10 +50,11 @@ It renders the `_table` partial to display details about the resources.
 </header>
 
 <section class="main-content__funnel">
-  <%= render "funnel_stage", stage: FunnelStage.new(previous_cohort: [], cohort: resources, name: "Created") %>
-  <%= render "funnel_stage", stage: FunnelStage.new(previous_cohort: resources, cohort: resources.signed, name: "Signed") %>
-  <%= render "funnel_stage", stage: FunnelStage.new(previous_cohort: resources.signed, cohort: resources.faxed, name: "Faxed") %>
-  <%= render "funnel_stage", stage: FunnelStage.new(previous_cohort: resources.faxed, cohort: resources.emailed, name: "Emailed") %>
+  <%- all_resources = resources.unscoped %>
```

Comment:
administrate provides the `resources` value as a local, not as a helpper method, so we can't write helper methods to intercept it :(.